src/vs/workbench/parts/files/browser/fileActions.ts

```diff
@@ -1370,7 +1372,8 @@ export abstract class BaseSaveOneFileAction extends BaseSaveFileAction {
 		if (this.resource) {
 			source = this.resource;
 		} else {
-			source = toResource(this.editorService.getActiveEditorInput(), { supportSideBySide: true, filter: ['file', 'untitled'] });
+			// source = toResource(this.editorService.getActiveEditorInput(), { supportSideBySide: true, filter: ['file', 'untitled'] });
+			source = toResource(this.editorService.getActiveEditorInput(), { supportSideBySide: true });
```

Comment:
I believe we need to revisit all `toResource`-calls. A surprising large amount of calls are guarded by the `file` or `untitled` thing and I am not entirely sure why...

Reply:
@jrieken simple, only untitled and file schemes are writeable. Many assumptions are being made based on that. If we had a registry for this we could ditch that assumption in favour of the registry.

Reply:
I'd favour a property on the model instead of yet another registry. The *thing* creating a model should know if it can be saved or not. With that we could also honour readonly-files etc

Reply:
Not sure that works in all cases. E.g. now I can just have a resource (or editor input with a resource) and make a statement if the resource is editable or not. This e.g. can be used to control the number of actions we show in a context menu for a file that is there but not yet opened (e.g. no model loaded yet).

Reply:
Yeah, maybe not the model but inside the stat or one the model objects we get from the fs-service

Reply:
But not all resources are resolved, the "Open Editors" section (or quick open) just have a list of resources that are not resolved (neither as stat, nor as model).